package.json

```diff
@@ -10,17 +10,23 @@
     "test": "jest --watchAll"
   },
   "dependencies": {
+    "@lottiefiles/react-lottie-player": "^3.5.3",
     "date-fns": "^2.30.0",
     "eslint": "8.44.0",
     "eslint-config-next": "13.4.8",
     "leaflet": "^1.9.4",
     "leaflet-defaulticon-compatibility": "^0.1.2",
+    "lottie-web": "^5.12.2",
     "next": "13.4.8",
+    "proptypes": "^1.1.0",
```

Comment:
If you're not using `proptypes` best remove it.

Reply:
The react-datapicker documentation asked for proptypes, that's why I installed it.

Reply:
*react-datepicker

Reply:
Hm. Weird.
Since it doesn't hurt to have it installed, it doesn't affect the behavior of the app, let's leave it then. 👍🏼